ztp/source-crs/PerformanceProfile.yaml

```diff
@@ -8,6 +8,7 @@ spec:
   additionalKernelArgs:
   - "idle=poll"
   - "rcupdate.rcu_normal_after_boot=0"
+  - "crashkernel=256M"
```

Comment:
I would rather we put this in the 06-kdump-enable-master file and not here...  Which would take another edit to mcmaker :)

Reply:
That would be fine! Just remember that we need to make a master and worker, so maybe still use a generator script like sctp does to avoid duplication.